dotCMS/src/main/java/com/dotcms/concurrent/DotSubmitter.java

```diff
@@ -65,6 +65,18 @@
      */
     public int getActiveCount();
 
+    /**
+     * Gets the pool size configured
+     * @return int
+     */
+    public  int getPoolSize();
+
+    /**
+     * Gets the Max pool size configured
+     * @return int
+     */
+    public  int getMaxPoolSize();
```

Comment:
Codacy found an issue: [Unnecessary modifier 'public' on method 'getMaxPoolSize': the method is declared in an interface type](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6265363)